webapp-django/crashstats/crashstats/views.py

```diff
@@ -1143,6 +1151,79 @@ def crashes_per_user(request, default_context=None):
+    #
+    # But no number of "1.x" if that was what exploded into 1.1, 1.2, 1.3.
+    # So we need to put that back and manually sum the counts. So that
+    # eventually we end up with this instead:
+    #
+    #  version 1.x : 5700,   (=1234+3456+1010 as per the example)
+    #
+    # Note that we assume the throttle to be the same.
+    #
+
+    for version, sub_versions in exploded_version_numbers.items():
+        # First compress the adi_counts (i.e. opposite of exploding)
+        new = {}
+        for sub_version in sub_versions:
+            for date, (count, throttle) in adi_by_version[sub_version].items():
+                if date not in new:
+                    new[date] = [0, throttle]
+                new[date][0] += count
+            del adi_by_version[sub_version]
+        adi_by_version[version] = new
+
+        # Same idea, but let's deal with the SuperSearch results
+        #
+        # The `results['facets']` looks something like this:
+        #     {
+        #        'histogram_date': [
+        #            {
+        #                'count': 1234,
+        #                'facets': [
+        #                    {'count': 201, 'term': '1.0'},
+        #                    {'count': 196, 'term': '1.1'},
+        #                    {'count': 309, 'term': '1.2'},
+        #                    {'count': 991, 'term': '2.5'},
+        #                ],
+        #                'term': '2015-01-10T00:00:00'
+        #            },
+        #            ...
+        #
+        #        'version': [
+        #            {'count': 45234, 'term': '1.0'},
+        #            {'count': 39001, 'term': '1.1'},
+        #            {'count': 56123, 'term': '1.2'},
+        #            {'count': 90133, 'term': '2.5'},
+        #            ...
+        #
+        # Our job, now is to compress those little buckets by how
+        # we exploded the versions before.
+        #
+        # Note, that we have to do this to the 'version' key too
+        #
+        histogram = results['facets']['histogram_date']
+        for date_cluster in histogram:
+            new = {'count': 0, 'term': version}
+            for facet_cluster in list(date_cluster['facets']['version']):
+                if facet_cluster['term'] in sub_versions:
+                    # it needs to be removed and its count added to 'new'
+                    new['count'] += facet_cluster['count']
+                    date_cluster['facets']['version'].remove(facet_cluster)
+            date_cluster['facets']['version'].append(new)
+        # Lastly for the "summary facet"
+        new_summary_cluster = {'count': 0, 'term': version}
+        for facet_cluster in list(results['facets']['version']):
+            if facet_cluster['term'] in sub_versions:
+                new_summary_cluster['count'] += facet_cluster['count']
+                results['facets']['version'].remove(facet_cluster)
+        results['facets']['version'].append(new_summary_cluster)
```

Comment:
What this thing above means is that [this hack](https://github.com/mozilla/socorro/blob/f1022e9ef9d4df19b1e674ab6b1199d4a561797f/socorro/lib/search_common.py#L385) is "no longer needed". I put it in apostrophes because it's not needed by the Crashes per User report any more.